README.md

```diff
@@ -15,14 +15,18 @@ The `--url` parameter is used for 2 reasons:
 
 If you have a local copy of the ISO, you can point to it with the `--iso` parameter to avoid having `ubuntu-server-netboot.py` download an extra copy. Just be sure that `--iso` and `--url` point to the same version of the ISO.
 
+Optionally, you can place `--autoinstall-url` to tell the netbooting process to enable subiquity automation. See [our autoinstall example](./autoinstall/README.md) and [the autoinstall and Automated Server Installs
+Introduction of Ubuntu Server guide](Automated Server Installs Introduction) for more details.
+
 You can also add additional kernel command line arguments (e.g. `"console=ttyS0"`) to the generated configuration files using the `--extra-args` parameter.
 
-## Usage
+## Usage of the Generated Files
 Copy the files generated under the interim folder `/tmp/tmpxxx/ubuntu-installer/`
-to your tftp root folder for netboot, for example `/var/lib/tftpboot`:
+to your tftp root folder for netboot, for example `/srv/tftp` or `/var/lib/tftpboot`.
+You may check your tftpd configuration of the root directory via `/etc/default/tftpd-hpa`. Let's copy:
 
 ```
-$ sudo cp /tmp/tmpxxx/ubuntu-installer/* /var/lib/tftpboot/
+$ sudo cp -r /tmp/tmpxxx/ubuntu-installer/* /srv/tftp
```

Comment:
While I agree with this, can you split it into a separate commit (maybe combined w/ the other `Usage` changes?)

Reply:
+1. Will do in the upcoming PR update.